src/index.html

```diff
@@ -7,8 +7,73 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+  
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img 
+          class="logo"
+          src="./images/logo.png"
+          alt="logo_image"
+        >
+      </a>
+      
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#apple" class="is-active nav_link">
+              apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#samsung" class="nav_link">
+              samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#smartphones" class="nav_link">
+              smartphones
+            </a>
+          </li>
+          <li class="nav_item">
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav_list">
          <li class="nav_item">
            <a href="#apple" class="is-active nav_link">
              apple
            </a>
          </li>
          
          <li class="nav_item">
            <a href="#samsung" class="nav_link">
              samsung
            </a>
          </li>
          
          <li class="nav_item">
            <a href="#smartphones" class="nav_link">
              smartphones
            </a>
          </li>
          
          <li class="nav_item">
```
dont forget to add space between one level blocks